### DISCUSSION TITLE
How to Configure a Download-Only Table in Dotmim.Sync?

### DISCUSSION THREAD

**Discussion by herczegzoltan** [Mimetis/Dotmim.Sync#1283] [Q&A] [answered]
I have successfully synced multiple tables, but I need one specific table to be download-only. I looked at the sample project, but it appears to be empty.

[https://github.com/Mimetis/Dotmim.Sync/blob/master/Samples/SqliteDownloadOnlySyncProvider/SqliteDownloadOnlySyncProvider.cs](
)

Could someone provide an example?

**Comment by herczegzoltan** [comment]
This setup uses a Maui + Web API. Here's what I did on the client side:

```
var setup = new SyncSetup("TestTable1", "TestTable2");
setup.Tables["TestTable2"].SyncDirection = SyncDirection.DownloadOnly;
var agent = new SyncAgent(clientProvider, serverOrchestrator, options);
var progress = new SynchronousProgress<ProgressArgs>(args => Console.WriteLine($"{args.ProgressPercentage:p}:\t{args.Message}"));

```

I removed all migrations and deleted the existing tables on the server. Then, I added a new migration, and all tables were successfully created.

After cleaning the database on the client and running the app, the sync method throws the following exception:

```
Dotmim.Sync.SyncException: Seems you are trying another Setup than what is stored in your server scope database. 
You have already made a sync with a setup that has been stored in the server.
```

What am I doing wrong?

**Reply by Mimetis** [reply]
Try a new scope name

**Reply by herczegzoltan** [reply]
Do you mean the download only tables needs different scope than the others?
like this:
https://dotmimsync.readthedocs.io/Scopes.html

Just because before I removed everything client local database file, server database and close all connection also got rid of the migrations. To start fresh and when I start the app it creates the tables but the above error is thrown.

**Reply by Mimetis** [reply, marked as answer]
The problem is that you have changed your `SyncSetup` (by adding S`yncDirection.DownloadOnly` to one table) **after** having made some syncs.
That's why you have this error.

If you are in testing mode, I suggest to start from scratch each time you are changing your setup
You can start with  a fresh backup from you database without any **DMS** related metadata
Or you can call a `DropAllAsync()` method to remove everything from **DMS** in your database, and start with a new setup
